templates/web/base/claims/confirmation.html

```diff
@@ -5,7 +5,7 @@
 <h1 class="govuk-heading-xl">Claim submitted</h1>
 
 [% IF auto_response %]
-    [% auto_response.text | staff_html_markup(auto_response.extra) %]
+    [% auto_response | staff_html_markup() %]
```

Comment:
This will mean HTML in the template is escaped rather than shown - I think we can safely pass in `({is_body_user:1})` here?